### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by solomon-negusse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,25 @@
...
 ## Setting up the AWS resources
 
 To setup the cluster cloud resources, use the following command:
 
 ```shell script
 cd ./terraform
-terraform init
-terraform plan
-CLOUDFLARE_API_KEY=<cloudflare api key> CLOUDFLARE_EMAIL=<cloudflare api key> terraform apply  -var-file=vars/core-dev.tfvarsemail
+export CLOUDFLARE_API_KEY=<cloudflare api key> CLOUDFLARE_EMAIL=<cloudflare api key>
```
Comment (tiagojsag) [inline README.md]:
```suggestion
export CLOUDFLARE_API_KEY=<cloudflare api key> CLOUDFLARE_EMAIL=<cloudflare account email address>
```

File: README.md [hunk 2]
```diff
@@ -102,23 +112,22 @@ Assuming your public key was sent to the bastion host during the setup process,
...
 
 - Copy the `kubectl_config` settings from above into your local `~/.kube/config`
 - Modify the `server: https://<random string>.gr7.us-east-1.eks.amazonaws.com` line by adding `:4433` at the end, so it looks like this: `server: https://<random string>.gr7.us-east-1.eks.amazonaws.com:4433` (you can pick a different port if you want)
-- Modify your local `/etc/hosts` to include the following line: `127.0.0.1  https://<random string>.gr7.us-east-1.eks.amazonaws.com`
-  
+- Modify your local `/etc/hosts` to include the following line: `127.0.0.1 <random string>.gr7.us-east-1.eks.amazonaws.com`
 
 ```shell script
-ssh -N -L 4433:DA387DC6CA64435B70B143F167D2E3C7.gr7.us-east-1.eks.amazonaws.com:443 ubuntu@ec2-3-92-73-248.compute-1.amazonaws.com
+ssh -N -L 4433:<random string>.gr7.us-east-1.eks.amazonaws.com:443 ubuntu@<bastion_hostname>
```
Comment (tiagojsag) [inline README.md]:
```suggestion
ssh -N -L 4433:<eks api endpoint url>:443 <bastion user>@<bastion hostname>
```

File: README.md [hunk 3]
```diff
@@ -102,23 +112,22 @@ Assuming your public key was sent to the bastion host during the setup process,
...
-### SSH tunnel 
+### SSH tunnel
 
 Perhaps the most practical way to connect to the cluster is by creating an SSH tunnel that connects a local port to the cluster's API port, through the bastion. For this to work, a few things are needed:
 
 - Copy the `kubectl_config` settings from above into your local `~/.kube/config`
 - Modify the `server: https://<random string>.gr7.us-east-1.eks.amazonaws.com` line by adding `:4433` at the end, so it looks like this: `server: https://<random string>.gr7.us-east-1.eks.amazonaws.com:4433` (you can pick a different port if you want)
-- Modify your local `/etc/hosts` to include the following line: `127.0.0.1  https://<random string>.gr7.us-east-1.eks.amazonaws.com`
-  
+- Modify your local `/etc/hosts` to include the following line: `127.0.0.1 <random string>.gr7.us-east-1.eks.amazonaws.com`
```
Comment (tiagojsag) [inline README.md]:
```suggestion
- Modify your local `/etc/hosts` to include the following line: `127.0.0.1 <eks api endpoint url>`
```

**Review by tiagojsag — APPROVED** [review]
@solomon-negusse be mindful that PRs should be made to dev at all times, not to production. Otherwise LGTM

**Comment by solomon-negusse** [comment]
Roger that, thanks.

**Pull request merged**